src/tracer/MOM_tracer_flow_control.F90

```diff
@@ -527,13 +543,17 @@ subroutine call_tracer_column_fns(h_old, h_new, ea, eb, fluxes, mld, dt, G, GV,
                                            evap_CFL_limit=evap_CFL_limit, &
                                            minimum_forcing_depth=minimum_forcing_depth, Hbl=Hbl)
     endif
-    if (CS%use_MARBL_tracers) &
-      call MARBL_tracers_column_physics(h_old, h_new, ea, eb, fluxes, dt, &
-                                        G, GV, US, CS%MARBL_tracers_CSp, tv, &
+    if (CS%use_MARBL_tracers) then
+      if ((.not. present(prediabatic_T)) .or. (.not. present(prediabatic_S))) &
+        call MOM_error(FATAL, 'Must pass prediabatic_T and prediabatic_S when using MARBL')
+      call MARBL_tracers_column_physics(h_old, ea, eb, fluxes, dt, &
+                                        G, GV, US, CS%MARBL_tracers_CSp, &
+                                        prediabatic_T, prediabatic_S, &
                                         KPP_CSp=KPP_CSp, &
                                         nonLocalTrans=nonLocalTrans, &
                                         evap_CFL_limit=evap_CFL_limit, &
                                         minimum_forcing_depth=minimum_forcing_depth)
+    end if
```

Comment:
MOM6 style guide says to use `endif` instead of `end if`.
There are 3 occurrences of `end if` in this PR.